api/inc/box_config.h

```diff
@@ -109,6 +109,9 @@ UVISOR_EXTERN void const * const main_cfg_ptr;
     \
     UVISOR_EXTERN const __attribute__((section(".keep.uvisor.cfgtbl_ptr"), aligned(4))) void * const box_name ## _cfg_ptr = &box_name ## _cfg;
 
+#define UVISOR_BOX_CFG_PTR_DECL(box_name) \
```

Comment:
How about we call this `UVISOR_BOX_EXTERN`? We know that internally it declares the box cfg ptr, but from the POV of the user it's just extern-declaring the box.